app/components/quiz/QuizAnswer.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react';
+import AppStore from 'store/appStore';
+import 'quiz.scss';
+import jquery from 'jquery';
 
 export default React.createClass({
   displayName: 'QuizAnswer',
 
   propTypes: {
-    answer: React.PropTypes.object.isRequired
+    answer: React.PropTypes.object.isRequired,
```

Comment:
I still need the `id` to pass back up to the QuizQuestion in the `this.props.answerClicked` method though